src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search--form search--form__big"
+      data-qa="big"
+    >
+      <label class="search--form__label search--form__label_big">
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class 'search--form__big' seems to represent a style rather than the content meaning.